libmproxy/console/common.py

```diff
@@ -263,6 +263,42 @@ def copy_flow_format_data(part, scope, flow):
                     raise ValueError("Unknown part: {}".format(part))
     return data, False
 
+def copy_as_curl_command(flow):
+
+    if flow.request.content is None or flow.request.content == CONTENT_MISSING:
+        return None, "Request content is missing"
+
+    headerString = ""
+    for k,v in flow.request.headers:
+      headerString += " -H \"" + k + ":" + v + "\" "
+
+    data = "curl" + " -X " + flow.request.method  + headerString + flow.request.scheme + "://" + flow.request.host + flow.request.path
```

Comment:
I thought about this a bit.  You're probably right, we can quote it, but i was testing a curl command with query params and when copy and pasting into the shell, it auto escapes '=' and '?' properly